### PR TITLE
[DCM-KIT] DCM Standard Add On #608

### PR DESCRIPTION
**Pull request by ClosedSourcerer**
## Description

This PR contains DCM-Kit content for release 24.08.

The following Issues are addressed: 
|Issue ID|Name|Structure|Content|
|--|--|--|--|
|[608](https://github.com/eclipse-tractusx/sig-release/issues/608)|[DCM-KIT] DCM Standard Add On Explanation|X|X|
|[585](https://github.com/eclipse-tractusx/sig-release/issues/585)|[DCM-KIT] Q&A Section|X|
|[587](https://github.com/eclipse-tractusx/sig-release/issues/587)|[DCM-KIT] Onboarding for DCM Access|X|

Content for 585 and 587 will brought in via seperate PRs.

Some highlites:
- Added missing license information (TRG.7.08)
- Introduced Swagger API documentation
- Restructured Kit
- Added sections for Standard Add Ons

(https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/981 AND https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/984 AND https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/985) REPLACE https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/969

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [ ] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [X] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

**Review by MWADCM — COMMENTED** [review]
Yeah!

**Review by jschu1 — APPROVED** [review]

File: docusaurus.config.js [hunk 1]
```diff
@@ -373,6 +373,79 @@ const config = {
...
+            specPath:
+              './openApi/dcm/WeekBasedMaterialDemand.yaml', // Path to designated spec file
+            outputDir:
+              './docs-kits/kits/DCM-Kit/development-view/plugin-generated-material-demand-api', // Output directory for generated .mdx docs
+            sidebarOptions: {
+              groupPathsBy: 'tag',
+            },
+          },
+
+          dcm_id_based_comment: {
```
Comment (nhaenis) [inline docusaurus.config.js:413]: Duplicate IDs, please remove

File: docusaurus.config.js [hunk 2]
```diff
@@ -373,6 +373,79 @@ const config = {
...
+          },
+          dcm_week_based_material_demand: {
+            specPath:
+              './openApi/dcm/WeekBasedMaterialDemand.yaml', // Path to designated spec file
+            outputDir:
+              './docs-kits/kits/DCM-Kit/development-view/plugin-generated-material-demand-api', // Output directory for generated .mdx docs
+            sidebarOptions: {
+              groupPathsBy: 'tag',
+            },
+          },
```
Comment (nhaenis) [inline docusaurus.config.js:448]: Duplicate IDs, please remove

Comment (ClosedSourcerer) [reply docusaurus.config.js:411]: Fixed via https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/981/commits/9fdc02290c2208358197e44c36bd3ff593e9f6a8

**Review by nhaenis — APPROVED** [review]

**Comment by ClosedSourcerer** [comment]
@LukasSchmetz 
@MWADCM 
Please take care of the Eclipse IP clearing process.

https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/?search=automotive.tractusx&sort=created_date&state=opened&first_page_size=20

https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-03

https://eclipse-tractusx.github.io/docs/oss/issues/#eclipse-gitlab-ip-issue-tracker

**Comment by LukasSchmetz** [comment]
> @LukasSchmetz @MWADCM Please take care of the Eclipse IP clearing process.
> 
> https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/?search=automotive.tractusx&sort=created_date&state=opened&first_page_size=20
> 
> https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-03
> 
> https://eclipse-tractusx.github.io/docs/oss/issues/#eclipse-gitlab-ip-issue-tracker

@danielmiehle @maximilianong @jSchuetz88: Can you please let us know what needs to be done and where? We have never done any Eclipse IP clearing process. If we got it right from the links above only a committer can open such an IP ticket. Many thanks for the support.

**Comment by danielmiehle** [comment]
@jSchuetz88: Hi Johann, could you clarify how to and who can start the IP-clearance process for the DCM?


fyi @Grand-Thibault @stephanbcbauer

**Comment by jSchuetz88** [comment]
> @jSchuetz88: Hi Johann, could you clarify how to and who can start the IP-clearance process for the DCM?
> 
> fyi @Grand-Thibault @stephanbcbauer

Based on TRG 7.03:
> If not, a project committer must engage with the IP Team to request an [IP review for Code Contributions](https://eclipse-tractusx.github.io/docs/oss/issues#eclipse-gitlab-ip-issue-tracker) by the IP Team before the contribution is pushed/merged.

@stephanbcbauer As I'm not a committer, can you help here?

**Comment by danielmiehle** [comment]
IP check created: https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/15750

File: docs-kits/kits/DCM-Kit/changelog.md [hunk 3]
```diff
@@ -13,17 +12,33 @@ toc_max_heading_level: 2
 
 All notable changes to this Kit will be documented in this file.
 
-## [1.0.0] - unreleased
+## [1.1.0] - unreleased
 
 ### Added
```
Comment (jSchuetz88) [inline docs-kits/kits/DCM-Kit/changelog.md:18]: I think the changelog misses the added "Onboarding for DCM Access"

**Comment by danielmiehle** [comment]
> IP check created: https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/15750

PR still under IP review. Just wrote a reminder.
fyi: @ClosedSourcerer

**Comment by jSchuetz88** [comment]
@ClosedSourcerer @danielmiehle @maximilianong The IP-check has been approved.

**Review by danielmiehle — APPROVED** [review]
LGTM - ready to merge!

**Review by maximilianong — APPROVED** [review]

**Pull request merged**